ios/architecture.md

```diff
@@ -1,34 +1,66 @@
 # Architecture
 
-In Futured, we enforce our customized version of architecture described as MVVM-C (Model-View-ViewModel Coordinator). Only occasionally a different architecture is used, if product requires it or as a way to experiment and explore.
+In Futured, we prefer our customized version of architecture described as MVVM-C (Model-View-ViewModel-Coordinator)
+when using UIKit.
+
+Our team is used to it. This means it dramatically improves our time to kick-off a project and maintain its development.
+We can easily substitute developers. Onboarding of developers on a new project is faster too.
+We only occasionally use a different architecture, if the product requires it or as a way to experiment and explore.
```

Comment:
Taktiez pouzivame inu architekturu ked preberame projekt od niekoho ineho (sensorsafe, wavin..)

Reply:
To mi příjde celkem očividné, že když pokračujeme v převzatém projektu nemá naši architekturu. Mám to pro upřesnění připsat?